src/modules/profile/actions.js

```diff
@@ -30,19 +37,33 @@ export const finishedFetchingUserInfo = info => {
   }
 }
 
-export const fetchUserInfo = apiToken => {
+export const toggleMenu = toggle => {
```

Comment:
I think it's a bit of overkill to manage the toggle state of a menu in redux. I would do this locally in a component, much simpler